apis/nucleus/src/components/listbox/hooks/useOnTheFlyModel.jsx

```diff
@@ -6,12 +6,17 @@ export default function useOnTheFlyModel({ app, fieldIdentifier, stateName, opti
   const [fieldDef, setFieldDef] = useState('');
   const [isFetching, setIsFetching] = useState(true);
   const [model, setModel] = useState();
+  const [title, setTitle] = useState(options.title);
+  const setFallbackTitle = (fallbackTitle) => {
+    setTitle((currentTitle) => currentTitle ?? fallbackTitle);
+  };
```

Comment:
I expect this change to break the ability to update the title by updating options.title
(but I have not tested)

Reply:
hmm... do we even have the possibility to update options without re-rendering the listbox completely? How/what scenario should we do to test that?

Or maybe you have a suggestion on how we should re-write this to avoid that problem (that might only exist in theory at the moment...) @T-Wizard ?

Reply:
if you can not update the options then it is not a problem (or at lest not now)
and even if you can update the options it could be a limitation we can live with.

as for re-write suggestion, mabe something like this:

```javascript
const [fallbackTitle, setFallbackTitle] = useState();
const title = options.title ?? fallbackTitle;
```